src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -490,7 +491,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		this._register(toDisposable(() => this._dndObserver?.dispose()));
 
 		this._containerReadyBarrier = new AutoOpenBarrier(Constants.WaitForContainerThreshold);
-		this._attachBarrier = new AutoOpenBarrier(1000);
+		this._attachBarrier = new AutoOpenBarrier(3000);
```

Comment:
without this change from 1000 -> 3000, an error is thrown because the container is not yet ready so `xtermElement` is udf

Reply:
It's also only used on TerminalInstance.focus. Is this being confused with _containerReadyBarrier?

Reply:
yes, this was not needed and was the wrong thing. reverting